app/models/host.rb

```diff
@@ -22,7 +22,43 @@ class Host < ApplicationRecord
   validates :name, presence: true
   validates :account, presence: true
 
+  def in_inventory
+    host_in_inventory(id)
+  end
```

Comment:
I really liked this at first because of how convenient it was, but I think there may be an issue related to `current_user` I discussed already. Because the HostInventoryAPI requires a user account, I'm not sure we can ever move this sort of thing into the model directly like this.